go/install/install.go

```diff
@@ -273,3 +276,41 @@ type CommonLsofResult struct {
 	PID     string
 	Command string
 }
+
+func fileContainsWord(filePath, searchWord string) bool {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return false
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), searchWord) {
+			return true
+		}
+	}
+	return false
+}
+
+func LastModifiedMatchingFile(dirPath string, fileNameMatch string, fileContentMatch string) (res *string) {
```

Comment:
Minor code clarity thing. I find it harder to read this since the values `dirPath="${TEMP}", fileNameMatch="MSI", fileContentMatch="Keybase"` are far away at the callsite. For tests dirPath needs to be an arg, but the other two could be private constants.

Reply:
agreed. i collapsed the first two parameters into a single one that's just a pattern, but left all of the parameters as injectable. private constant feels a little lazier to me, and it binds this functionality to this one usecase forever. and that doesn't feel necessary to me.